### PR TITLE
update repositories

### PR DESCRIPTION
**Pull request by WaterLily**

### CODE REVIEW

File: .idea/jarRepositories.xml [hunk 1]
```diff
@@ -26,25 +26,15 @@
       <option name="name" value="MavenRepo" />
       <option name="url" value="https://repo.maven.apache.org/maven2/" />
     </remote-repository>
-    <remote-repository>
```
Comment (WaterLily) [inline .idea/jarRepositories.xml:29]: This is a test comment.

File: .idea/jarRepositories.xml [hunk 2]
```diff
@@ -26,25 +26,15 @@
...
-      <option name="id" value="C:\Users\jpham\AppData\Local\Android\Sdk\extras\google\m2repository" />
-      <option name="name" value="C:\Users\jpham\AppData\Local\Android\Sdk\extras\google\m2repository" />
-      <option name="url" value="file:/$USER_HOME$/AppData/Local/Android/Sdk/extras/google/m2repository" />
```
Comment (WaterLily) [inline .idea/jarRepositories.xml:32]: this is another test comment

File: .idea/jarRepositories.xml [hunk 3]
```diff
@@ -26,25 +26,15 @@
...
-    <remote-repository>
-      <option name="id" value="C:\Users\jpham\AppData\Local\Android\Sdk\extras\m2repository" />
-      <option name="name" value="C:\Users\jpham\AppData\Local\Android\Sdk\extras\m2repository" />
-      <option name="url" value="file:/$USER_HOME$/AppData/Local/Android/Sdk/extras/m2repository" />
-    </remote-repository>
     <remote-repository>
       <option name="id" value="MavenLocal" />
       <option name="name" value="MavenLocal" />
-      <option name="url" value="file:/$USER_HOME$/.m2/repository" />
+      <option name="url" value="file:/$MAVEN_REPOSITORY$" />
```
Comment (jphampton) [inline .idea/jarRepositories.xml:32]: This is a test comment.

Comment (jphampton) [reply .idea/jarRepositories.xml:32]: This is a test reply.

**Review by WaterLily — COMMENTED** [review]
submitting my review.

File: .idea/jarRepositories.xml [hunk 4]
```diff
@@ -26,25 +26,15 @@
...
-    <remote-repository>
-      <option name="id" value="C:\Users\jpham\AppData\Local\Android\Sdk\extras\google\m2repository" />
-      <option name="name" value="C:\Users\jpham\AppData\Local\Android\Sdk\extras\google\m2repository" />
-      <option name="url" value="file:/$USER_HOME$/AppData/Local/Android/Sdk/extras/google/m2repository" />
-    </remote-repository>
-    <remote-repository>
-      <option name="id" value="C:\Users\jpham\AppData\Local\Android\Sdk\extras\m2repository" />
-      <option name="name" value="C:\Users\jpham\AppData\Local\Android\Sdk\extras\m2repository" />
-      <option name="url" value="file:/$USER_HOME$/AppData/Local/Android/Sdk/extras/m2repository" />
-    </remote-repository>
```
Comment (jphampton) [inline .idea/jarRepositories.xml:38]: This is a test comment to begin a review.

**Review by jphampton — COMMENTED** [review]
This is a test comment to finish the review.

File: .idea/jarRepositories.xml (hunk 2 above)
Comment (jphampton) [reply .idea/jarRepositories.xml:32]: This is a test reply

Comment (WaterLily) [reply .idea/jarRepositories.xml:32]: thanks for the thorough feedback.

File: .idea/jarRepositories.xml (hunk 4 above)
Comment (jphampton) [inline .idea/jarRepositories.xml:38]: Test - remove this line!

**Review by jphampton — APPROVED** [review]
Nevermind, it's fine the way it was.

File: .idea/jarRepositories.xml [hunk 5]
```diff
@@ -26,25 +26,15 @@
...
     <remote-repository>
-      <option name="id" value="C:\Users\jpham\AppData\Local\Android\Sdk\extras\android\m2repository" />
-      <option name="name" value="C:\Users\jpham\AppData\Local\Android\Sdk\extras\android\m2repository" />
-      <option name="url" value="file:/$USER_HOME$/AppData/Local/Android/Sdk/extras/android/m2repository" />
+      <option name="id" value="MavenLocal" />
+      <option name="name" value="MavenLocal" />
+      <option name="url" value="file:/$MAVEN_REPOSITORY$/" />
     </remote-repository>
   </component>
 </project>
```
Comment (jphampton) [inline .idea/jarRepositories.xml:40]: Test - change this line, please!

**Review by jphampton — CHANGES_REQUESTED** [review]
please change

**Review by jphampton — APPROVED** [review]

**Pull request merged**